spec/jekyll-feed_spec.rb

```diff
@@ -239,9 +239,9 @@
     end
 
     it "correctly adds the baseurl to the posts" do
-      expect(contents).to match /http:\/\/example\.org\/bass\/2014\/03\/04\/march-the-fourth\.html/
-      expect(contents).to match /http:\/\/example\.org\/bass\/2014\/03\/02\/march-the-second\.html/
-      expect(contents).to match /http:\/\/example\.org\/bass\/2013\/12\/12\/dec-the-second\.html/
+      expect(contents).to match /http:\/\/example\.org\/bass\/updates\/2014\/03\/04\/march-the-fourth\.html/
```

Comment:
not related to this PR.. but IMO enforcing the `percent_r` literal on the test files will avoid the need to escape the slashes here..

/cc @DirtyF

Reply:
Yeah. Given the time, I'd also love to add unit tests to the generator and meta tag (rather than just relying on integration tests).